seacrowd/sea_datasets/phomt/phomt.py

```diff
@@ -0,0 +1,139 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{PhoMT,
+title     = {{PhoMT: A High-Quality and Large-Scale Benchmark Dataset for Vietnamese-English Machine Translation}},
+author    = {Long Doan and Linh The Nguyen and Nguyen Luong Tran and Thai Hoang and Dat Quoc Nguyen},
+booktitle = {Proceedings of the 2021 Conference on Empirical Methods in Natural Language Processing},
+year      = {2021},
+pages     = {4495--4503}
+}
+"""
+
+_DATASETNAME = "phomt"
+
+
+_DESCRIPTION = """\
+PhoMT is a high-quality and large-scale Vietnamese-English parallel dataset of 3.02M sentence pairs, which is 2.9M
+pairs larger than the benchmark Vietnamese-English machine translation corpus IWSLT15. This is the first large-scale
+Vietnamese-English machine translation study.
+"""
+
+_LANGUAGES = ["vie", "eng"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LOCAL = True
+
+_HOMEPAGE = "https://github.com/VinAIResearch/PhoMT"
+
+_LICENSE = Licenses.MIT.value
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+MAP_LANG = {"en": "eng", "vi": "vie"}
+
+
+def seacrowd_config_constructor(src_lang, tgt_lang, schema, version):
+    if src_lang == "" or tgt_lang == "":
+        raise ValueError(f"Invalid src_lang {src_lang} or tgt_lang {tgt_lang}")
+
+    if schema not in ["source", "seacrowd_t2t"]:
+        raise ValueError(f"Invalid schema: {schema}")
+
+    return SEACrowdConfig(
+        name="phomt_{src}_{tgt}_{schema}".format(src=src_lang, tgt=tgt_lang, schema=schema),
```

Comment:
Please also change the config names to `phomt_eng_vie_source` and `phomt_eng_vie_seacrowd_t2t`.